### ISSUE TITLE
Swipe to switch tabs

### ISSUE THREAD

**Issue by mvgorcum** [vector-im/riotX-android#422]
I think most people expect to switch to different tabs by swiping left to right.

**Comment by bmarty** [comment]
Yes, this is a possible UX. But maybe we will using swipe gesture to handle quick action on room as it is done on message (for starting a reply). This has to be decided.
CC @nadonomy

**Comment by baschi29** [comment]
I'm always happier when swiping brings me to a different tab than when it does something to the chats. Switching between tabs is just something I do a lot more often and beeing able to swipe makes it feel super natural and fast.
